static/etc/udev/rules.d/90-fixrtc.rule

```diff
@@ -0,0 +1,2 @@
+# when rtc0 is added run fixrtc
+SUBSYSTEM=="rtc", KERNEL=="rtc0", RUN+= "/usr/bin/fixrtc"
```

Comment:
Should this be
```suggestion
SUBSYSTEM=="rtc", KERNEL=="rtc*", RUN+= "/usr/bin/fixrtc"
```
in case there are more RTC devices?

Reply:
I think this makes sense, I will change it